metrics-operator/config/certmanager/certificate.yaml

```diff
@@ -0,0 +1,39 @@
+# The following manifests contain a self-signed issuer CR and a certificate CR.
+# More document can be found at https://docs.cert-manager.io
+# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
+apiVersion: cert-manager.io/v1
```

Comment:
the files in `config/certmanager` can be removed as we do the certificate injection with our own cert-manager, based on the `keptn.sh/inject-cert` label